### PR TITLE
Add page for Jordan Ashley

### PR DESCRIPTION
**Pull request by j-s-ashley**
- creates fellows-2024 directory in pages/fellows
- creates fellows-2024 directory in assets/pdf
    - includes project proposal for page
- creates fellows-2024 directory in assets/images/team
     - includes image for page

### CODE REVIEW

**Comment by j-s-ashley** [comment]
@matthewfeickert Here's the updated pull request!

File: _data/universities/utk.yml
Comment (matthewfeickert) [inline _data/universities/utk.yml]: The `_data/` directory is for meant for institute researchers/staff, so this isn't needed for Fellows (and I think the CI is throwing an error as it is expecting additional files to exist given this university file) so this can get deleted.

```suggestion
```

File: pages/fellows/2024/JordanAshley.md
Comment (matthewfeickert) [inline pages/fellows/2024/JordanAshley.md:21]: @rct225 just updated the Fellow new page docs (https://iris-hep.org/docs/fellow) but as this PR would beat the upload of all the Fellow proposal documents I'll have to defer to him on how the proposal should be named. I assume it will be something along the lines of

```
/assets/pdf/fellows-2024/IRIS001-proposal-Jordan-Ashley.pdf
```

but I'll need him to confirm.

File: pages/fellows/2024/JordanAshley.md
Comment (matthewfeickert) [inline pages/fellows/2024/JordanAshley.md]: Can shorten this up slighlty.

```suggestion
    Repackage the EnergyFlow and Wasserstein for PyPI and conda-forge using modern build tools.
```

**Review by matthewfeickert — CHANGES_REQUESTED** [review]
Thanks @j-s-ashley! Overall looks good, though I gave some small suggested edits and I think we'll need @rct225 to give some additional information on the proposal naming scheme.

**Comment by matthewfeickert** [comment]
@j-s-ashley one more thing. The photo you added is a bit large in terms of file size. The newly [updated docs](https://iris-hep.org/docs/fellow) as for something smaller and in the sub 1MB size range.

> Add a photo named First-Last.jpg or .png to the assets/images/team/fellows-2024 folder. It should be 320x240 pixels and in general should be 150kB or less.

Can you rescale the photo using some image editing software (and maybe compress it if required to decrease the storage size)?

File: pages/fellows/2024/JordanAshley.md
Comment (j-s-ashley) [reply pages/fellows/2024/JordanAshley.md]: I implemented the change but also removed "the" after "Repackage."

File: pages/fellows/2024/JordanAshley.md
Comment (rct225) [reply pages/fellows/2024/JordanAshley.md:21]: The name is fine for now.  There's more likely to be a collision by renaming it now since they will likely be bulk uploaded at some point.  I appreciate @j-s-ashley staying ahead of the game, so no need to change it.  Everything else looks OK.

**Comment by matthewfeickert** [comment]
> The name is fine for now. There's more likely to be a collision by renaming it now since they will likely be bulk uploaded at some point.

Thanks for clarification and some cleanup, @rct225. I think this looks good now with the exception of the image size.

**Review by matthewfeickert — APPROVED** [review]
Thanks @j-s-ashley. LGTM! :shipit:

**Comment by rct225** [comment]
@j-s-ashley Looks OK.  I'll merge once the change to the photo @matthewfeickert requested is complete.

**Comment by matthewfeickert** [comment]
> I'll merge once the change to the photo @matthewfeickert requested is complete.

That's in now, and under 1 MB, so I'll squash and merge this once the CI is done.

**Pull request merged**
